JS/Lesson15/sinan/script.js

```diff
@@ -116,14 +129,49 @@ function checkIfStringIsUpperCase(stringToCheck) {
 	}
 }
 
-checkIfStringIsUpperCase('Write a program that takes in');
-checkIfStringIsUpperCase('write a program that takes in');
+checkIfStringIsUpperCase('Exercise 4.1:','Write a program that takes in');
+checkIfStringIsUpperCase('Exercise 4.2:','write a program that takes in');
 
 // 5. Write a program that takes in a number and checks if it's positive, negative, or zero using if else statement.
+function numberSign(number) {
+	if (number > 0){
+		return 'number is positive';
+	}
+	else if (number < 0){
+		return 'The number is negative';
+	}
+	else if (number === 0){ 
+		return 'The number is equal to zero';
+	}
+	else {
+		return 'NaN';
+	}
+}
+console.log('Exercise 5:',numberSign(-55));
+
 
 // 6. Write a program that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+function isLeapYear(year) {
```

Comment:
It's probably the exercise description that confused you a bit. When saying "if else" statement in JS, people usually mean any part or all parts of it (could be just `if (somethig) do something`, just like needed here).
I'd recommend to solve this problem in another way:
```
function isLeapYear(year) {
  if ((year % 4 === 0 && year % 100 !== 0) || year % 400 === 0) {
    return true;
  } else {
    return false;
  }
}
```

A negation is usually harder to read  and understand, so the solution above allow simplifying this code: `else if (!year %4 === 0){`